cartoframes/data/clients/sql_client.py

```diff
@@ -138,22 +171,26 @@ def describe(self, table_name, column_name):
         '''.format(','.join(stats), table_name)
         output = self.query(query, verbose=True)
         fields = output.get('rows')[0]
-        rows = [(key, '{:0.2e}'.format(fields[key])) for key in fields]
+        rows = [(key, '{:0.2e}'.format(fields[key])) for key in fields if fields[key] is not None]
         self._print_table(rows, padding=[5, 10])
         print('type: {}'.format(column_type))
 
-    def create_table(self, table_name, columns, cartodbfy=True):
+    def create_table(self, table_name, column_names, column_types, cartodbfy=True):
```

Comment:
Function LGTM, but adding two suggestions below in case you believe these would make sense. Otherwise please ignore :pray:

- Use a dictionary instead of two lists to pass the `column_names` and `column_types` parameters. This could be useful especially for large table schemas. 

- Use a collision strategy (e.g: `if_exists`) to avoid dropping the table if exists by default 

Including a slightly modified version just in case it was any useful, 

```python
def create_table(self, table_name, columns_dtypes, if_exists='fail', cartodbfy=True):
        """Create a table with a specific table name and columns.

        Args:
            table_name (str): name of the table.
            columns_dtypes (dict): dictionary with the column names and datatypes 
            if_exists (str): collision strategy if the table already exists in CARTO. 
            	Options are 'fail' or 'replace'. Default 'fail'.
            cartodbfy (bool, optional): convert the table to CARTO format.
                Default True. More info `here
                <https://carto.com/developers/sql-api/guides/creating-tables/#create-tables>`.
        Example:
            >>> sql.create_table('table_name', {'column1': 'text', column2: 'integer'})

        """
        if if_exists not in COLLISION_STRATEGIES:
        	raise ValueError("Please provide a valid if_exists value among {}".format(', '.join(COLLISION_STRATEGIES)))
        
        if not isinstance(columns_dtypes, dict):
        	raise ValueError("The column_dtypes parameter should be a dictionary of column names and datatypes (e.g: {'column_1': 'text'})")

        list_column_dtypes = ["{0} {1}".format(column_name, dtype) for column_name, dtype in columns_dtypes.items()]
        columns = ', '.join(list_column_dtypes)
        schema = self._context_manager.get_schema()
        query = '''
            BEGIN;
            {drop}
            {create}
            {cartodbfy}
            COMMIT;
        '''.format(
            drop='DROP TABLE IF EXISTS {};'.format(table_name) if if_exists == 'replace' else '',
            create='CREATE TABLE {0} ({1});'.format(table_name, columns),
            cartodbfy='SELECT CDB_CartoDBFyTable(\'{0}\', \'{1}\');'.format(
                schema, table_name) if cartodbfy else ''
        )
        return self.execute(query)
```

Reply:
The `COLLISION_STRATEGIES` constant would need to be added in order to test the code above

```python
COLLISION_STRATEGIES = ['fail', 'replace']
```

Reply:
Yep I thought about that and I kept it simple and similar to the implementation of `insert_table`. However, even if these utilities are simple, adding if_exists and the input dict looks better and is more useful, I'll change both `create_table` and `insert_table`. Thanks